homeassistant/components/broadlink/const.py

```diff
@@ -11,3 +11,28 @@
 
 SERVICE_LEARN = "learn"
 SERVICE_SEND = "send"
+
+A1_TYPES = ["a1"]
+MP1_TYPES = ["mp1"]
+RM_TYPES = [
```

Comment:
why don't use only "RM_TYPES" group?

Reply:
RM4 devices have their own class because they need special headers for communication.